include/aspect/particle/world.h

```diff
@@ -407,6 +407,15 @@ namespace aspect
         update_next_free_particle_index();
 
         /**
+         * Get a map between subdomain id and the neighbor index. In other words
+         * the returned map answers the question: Given a subdomain id, which
+         * neighbor (in terms of a contiguous number from 0 to n_neighbors)
+         * owns this subdomain?
```

Comment:
Would you mind changing this to say "which neighbor of the current processor's subdomain (in terms of ...)" to make clear that this is information that will be different on every processor.

This is a function that I think we ought to move to deal.II in the long term. But that can happen after this patch.